packages/components/Button/src/Button.tsx

```diff
@@ -110,9 +111,26 @@ export const Button = compose<ButtonType>({
         return (
           <Slots.root {...mergedProps} accessibilityLabel={label}>
             {buttonContent}
+            {button.state.focused && button.state.useTwoToneBorder && (
+              <Slots.focusInnerBorder
+                style={getFocusBorderStyle(button.state.height, button.state.width)}
+                accessible={false}
+                focusable={false}
+              />
+            )}
           </Slots.root>
         );
       }
     };
   },
 });
+
+const getFocusBorderStyleWorker = (height: number, width: number) => {
+  const adjustment = 2; // width of border * 2
```

Comment:
nit is the comment accurate w/ `*`?

Reply:
Yeah 2 is the width of the border twice.